csrc/scheduler/utils.h

```diff
@@ -31,6 +31,25 @@ namespace scheduler_utils {
 // but it's hard to get a better one.
 constexpr int64_t register_file_size_full = (int64_t)256 * 1024;
 constexpr int64_t register_file_size = register_file_size_full / 2;
+
+//! register file size allowed for persistent buffers in
```

Comment:
Looks like these parameters are only for the combined scheduler. Shouldn't they be moved to its own file?

On the other hand, I'd move `getSharedMemoryOverheadPerBlock` here as it isn't specific to any particular scheduler.

Reply:
make sense. Moved.